p2p/client.go

```diff
@@ -292,6 +368,14 @@ func (c *Client) setupPeerDiscovery(ctx context.Context) error {
 	return nil
 }
 
+func (c *Client) setupBlockSync(ctx context.Context) error {
+	blocksync := SetupBlockSync(ctx, c.Host, c.blockSyncStore, c.blockSyncReceived, c.logger)
+	c.blocksync = blocksync
+	go c.retrieveBlockSyncLoop(ctx)
```

Comment:
func should not be called setup imo since it also starts it

Reply:
updated to startBlockSync